pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -341,39 +343,81 @@ func (o *Options) Run() error {
 				fmt.Fprintf(o.Out, "Upstream: %s\n", cv.Spec.Upstream)
 			}
 			if len(cv.Status.Desired.Channels) > 0 {
-				fmt.Fprintf(o.Out, "Channel: %s (available channels: %s)\n", cv.Spec.Channel, strings.Join(cv.Status.Desired.Channels, ", "))
+				fmt.Fprintf(o.Out, "Channel: %s (available channels: %s)\n\n", cv.Spec.Channel, strings.Join(cv.Status.Desired.Channels, ", "))
 			} else {
-				fmt.Fprintf(o.Out, "Channel: %s\n", cv.Spec.Channel)
+				fmt.Fprintf(o.Out, "Channel: %s\n\n", cv.Spec.Channel)
```

Comment:
Ah, yes. I have no problem changing that. And...

>  it would be good to have blank lines associated with the incoming block instead of the outgoing block

Is definitely a valid point, and I would also agree it is a better way.

I have removed the previous blank lines from the `Channels` block, and added a blank line in: `fmt.Fprintf(o.Out, "\nRecommended updates:\n\n")`.
Fixed in the new commit 21f3636.